packages/builder/src/components/backend/DataTable/modals/CreateEditRow.svelte

```diff
@@ -29,6 +29,12 @@
       // Prevent modal closing if there were errors
       return false
     }
+
+    if (rowResponse.status === 500) {
```

Comment:
Maybe internally to the server we should catch the error and throw a 400 instead? Since this isn't an internal error and is an expected error given an invalid input.